command-injection/row_004_VisActor_VChart_typescript-json-schema.ts

```
@@ -0,0 +1,1787 @@
+          this.getClassDefinition(typ, definition);
+        }
+      }
+    }
+
+    if (this.recursiveTypeRef.get(fullTypeName) === definition) {
+      this.recursiveTypeRef.delete(fullTypeName);
+      // If the type was recursive (there is reffedDefinitions) - lets replace it to reference
+      if (this.reffedDefinitions[fullTypeName]) {
+        const annotations = Object.entries(returnedDefinition).reduce((acc, [key, value]) => {
+          if (annotationKeywords[key] && typeof value !== undefined) {
+            acc[key] = value;
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)